conduit/telemetry/telemetryCommon.go

```diff
@@ -0,0 +1,41 @@
+package telemetry
+
+import (
+	"time"
+
+	"github.com/opensearch-project/opensearch-go"
+)
+
+// Config represents the configuration of Telemetry logging
+type Config struct {
+	Enable    bool
+	SendToLog bool
+	URI       string
+	Name      string
+	GUID      string
+	Index     string
+	UserName  string
+	Password  string
+}
+
+// Event represents a single event to be emitted to OpenSearch
+type Event struct {
```

Comment:
Could you add the version string?

Reply:
good point, added conduit LongVersion() string to the event and fixed existing tests: [234ca36](https://github.com/algorand/conduit/pull/74/commits/234ca3690cf169c537c9d9f10b90c430fd728a52)